src/Sfa.Tl.ResultsAndCertification.Models/Contracts/AdminDashboard/ReviewChangeRequest.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Sfa.Tl.ResultsAndCertification.Common.Enum;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Sfa.Tl.ResultsAndCertification.Models.Contracts.AdminDashboard
+{
+    public  class ReviewChangeRequest
```

Comment:
Why don't we have independent classes for each change types? Are we going to be updating this class each time we include a new change? 

I think we should have a base class with the common fields and then a class for each change type

Reply:
created independent classes